android-appcoins-billing/src/main/java/com/appcoins/sdk/billing/payasguest/IabView.java

```diff
@@ -15,8 +15,11 @@
 
   void redirectToWalletInstallation(Intent intent);
 
+  /**
+   * Pass a non-empty uid if you want to resume a transaction. Otherwise send null or empty
+   */
```

Comment:
I think you should create 2 different methods: 1 to resume a purchase, other to start it instead of using a null argument